docs/user-guide/postinstall.md

```diff
@@ -124,3 +124,80 @@ To get a detailed log for Mysterium node service, you can use the following comm
 ```
 sudo journalctl -u mysterium-node.service
 ```
+
+## Built-in UI and Node LAN discovery
+
+It is possible to start a Mysterium node with a built-in UI server to do the basic configuration.
+
+The `--ui.enable` flag should be used to start a node with built-in UI.
```

Comment:
Do we really need this switch? Why can't we simply start with UI?

Reply:
To get rid of it we need to do conditional compiling to exclude the whole ui from mobile devices.